rllib/examples/env/bandit_envs_recommender_system.py

```diff
@@ -143,9 +143,8 @@ def step(self, action):
 
         reward = 0.0
         if which_clicked < self.slate_size:
-            # Reward is 1.0 - regret if clicked. 0.0 if not clicked.
```

Comment:
Why did we remove this comment?

Reply:
I felt that it was very obvious and did not need a comment!

Reply:
Please, let's not remove comments that are already there! A lot of things make sense when one reads the code, but might be unclear when one just skims through things.

Also, here, we should explain, where the magic new 100.0 value comes from.